src/components/Shared/LinePlot/LinePlotComponent.tsx

```diff
@@ -662,6 +662,9 @@ export class LinePlotComponent extends React.Component<LinePlotComponentProps> {
             if (key === "colormapScaling") { // TODO: remove this to fully support multiple lines
                 return;
             }
```

Comment:
This part was also for preventing exporting the colormap scaling line to the histogram tsv file. Maybe we could add `noExport: true` to the colormap scaling line and delete these lines?

Reply:
yes, thanks. I have make the code clearer as you suggested.